spring-kafka/src/main/java/org/springframework/kafka/requestreply/ReplyingKafkaTemplate.java

```diff
@@ -85,15 +86,20 @@
 
 	private boolean sharedReplyTopic;
 
+	private Function<ProducerRecord<K, V>, CorrelationKey> correlationStrategy =
```

Comment:
Not related to your change, but can we already resolve a TODO in this class on the `TemplateRequestReplyFuture` ?